.travis.yml

```diff
@@ -6,7 +6,10 @@ cache:
 notifications:
   email: false
 node_js:
+  - 4
+  - 5
```

Comment:
agree, although 90% of the time it will work for all of those, at the end of the day it'll become a maintenance burden if we explicitly commit to all of those